Palace/Accounts/Library/AccountsManager.swift

```diff
@@ -304,9 +304,7 @@ let currentAccountIdentifierKey  = "TPPCurrentAccountIdentifier"
 
     if self.accounts().isEmpty || TPPConfiguration.customUrlHash() != nil {
       loadCatalogs(completion: completion)
-    } else {
-      completion?(true) //Called when resetting a custom registry
```

Comment:
This was being called when setting updating accounts, even for normal accounts, and was triggering an infinite loop. No longer need this line because we won't attempt to reset the account after clearing a custom registry.